src/axom/slic/docs/sphinx/sections/slic_macros.rst

```diff
@@ -0,0 +1,298 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/slic_macros:
+
+Slic Macros
+------------
+
+Slic provides a set of convenience macros that can be used to streamline
+logging within the `Axom Toolkit`_ and may be used within any application
+using Slic.
+
+.. note::
+
+  The :ref:`sections/slic_macros` are not the only interface
```

Comment:
Would it be better to move the slic macros section to an appendix "Slic macros used in Axom" and put a section in here showing the basic machanics of wrapping Slic API calls in macros? You can use the slic macros in Axom to illustrate in code examples and refer to the appendix to see the sort of macros applications may find useful.

My point is that you start this section by saying: Here are some macros we use in Axom, but applications don't use these. They write their own. As an interested user at this point, I would like to see how I can wrap Slic API calls in macros, not a detailed description of stuff that you tell me I probably won't use.